resources/manifests/kubelet-pod-checkpointer-psp-role-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: kubelet-pod-checkpointer-psp
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
```

Comment:
`s / Role / ClusterRole / g`

Reply:
Leaving it as Role is fine now as per our discussion on Slack